src/interface/countrytechnologyview.txt

```diff
@@ -708,6 +708,63 @@ new interface_file countrytechnologyview = {
 			scripted = yes
 		}
 		
+		instantTextBoxType = {
+			name = "survey_idea_army_text"
+			position = { x = (300 + 128 - 118) y = (128 + 148 - 155 - 6) }
```

Comment:
what? U didn't want to do math?

Reply:
I mean who would, just let arc handle it.

Reply:
well it looks less elegant ,wouldn't u agree?

Reply:
I mean kinda, but velocity is more important in this situation, as the position of elements can keep getting adjusted.

Reply:
Fair enough